### PR TITLE
Minor improvements to documentation/Dockerfile

### PR DESCRIPTION
**Pull request by dewhurstwill**

### CODE REVIEW

File: Examples/kubernetes-fluentd-daemonset.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+          value: "0"
+        resources:
+          limits:
+            memory: 200Mi
+          requests:
+            cpu: 100m
+            memory: 200Mi
+        volumeMounts:
+        - name: plugin-volume
+          mountPath: /fluentd/plugins/out_lm.rb
```
Comment (ahsan13jan) [inline Examples/kubernetes-fluentd-daemonset.yaml:39]: This does not explain where does the out_lm.rb comes from.
We don't want user to copy this file rather we want them to install the plugin and use.

Comment (dewhurstwill) [reply Examples/kubernetes-fluentd-daemonset.yaml:39]: This is directly from the LM Logs Beta documentation.
The documentation notes out_lm.rb is added as a configmap then the daemonset will just read the ConfigMap

Comment (ahsan13jan) [reply Examples/kubernetes-fluentd-daemonset.yaml:39]: Yes, I know but we will update that.

Comment (ahsan13jan) [reply Examples/kubernetes-fluentd-daemonset.yaml:39]: I'll prioritze it, to simplify that the deployment.

Comment (ahsan13jan) [reply Examples/kubernetes-fluentd-daemonset.yaml:39]:
```
apiVersion: apps/v1
kind: DaemonSet
metadata:
  annotations:
    deprecated.daemonset.template.generation: "1"
  creationTimestamp: "2020-08-25T14:27:52Z"
  generation: 1
  labels:
    app.kubernetes.io/instance: lm-k8s-fluent
    app.kubernetes.io/managed-by: Helm
    app.kubernetes.io/name: lm-k8s-fluent
    app.kubernetes.io/version: 1.16.0
    helm.sh/chart: lm-k8s-fluent-0.1.0
  name: lm-k8s-fluent
  namespace: kube-system
  resourceVersion: "177986"
  selfLink: /apis/apps/v1/namespaces/kube-system/daemonsets/lm-k8s-fluent
  uid: 16da50c5-5ba0-4082-bbcc-88a0d21d2811
spec:
  revisionHistoryLimit: 10
  selector:
    matchLabels:
      app.kubernetes.io/instance: lm-k8s-fluent
      app.kubernetes.io/name: lm-k8s-fluent
  template:
    metadata:
      creationTimestamp: null
      labels:
        app.kubernetes.io/instance: lm-k8s-fluent
        app.kubernetes.io/name: lm-k8s-fluent
    spec:
      containers:
      - image: lmmuhammadahsan/lm-logs-k8s-fluentd:latest
        imagePullPolicy: Always
        name: lm-k8s-fluent
        resources: {}
        securityContext: {}
        terminationMessagePath: /dev/termination-log
        terminationMessagePolicy: File
        volumeMounts:
        - mountPath: /fluentd/etc
          name: fluentconf
        - mountPath: /var/log
          name: varlog
        - mountPath: /var/lib/docker/containers
          name: varlibdockercontainers
          readOnly: true
      dnsPolicy: ClusterFirst
      restartPolicy: Always
      schedulerName: default-scheduler
      securityContext: {}
      serviceAccount: lm-k8s-fluent
      serviceAccountName: lm-k8s-fluent
      terminationGracePeriodSeconds: 30
      tolerations:
      - operator: Exists
      volumes:
      - configMap:
          defaultMode: 420
          name: lm-k8s-fluent
        name: fluentconf
      - hostPath:
          path: /var/log
          type: ""
        name: varlog
      - hostPath:
          path: /var/lib/docker/containers
          type: ""
        name: varlibdockercontainers
  updateStrategy:
    rollingUpdate:
      maxUnavailable: 1
    type: RollingUpdate
status:
  currentNumberScheduled: 1
  desiredNumberScheduled: 1
  numberAvailable: 1
  numberMisscheduled: 0
  numberReady: 1
  observedGeneration: 1
  updatedNumberScheduled: 1
~
```

Comment (ahsan13jan) [reply Examples/kubernetes-fluentd-daemonset.yaml:39]: Something like this. Here I have used my personal repo. I need to fix it with LM account
**lmmuhammadahsan/lm-logs-k8s-fluentd:latest**

**Pull request closed without merging**